src/bartholomew.rs

```diff
@@ -114,6 +147,14 @@ pub fn render(req: Request) -> Result<Response> {
                         .clone()
                         .unwrap_or_else(|| DEFAULT_CONTENT_TYPE.to_owned());
 
+                    if is_dynamic_template {
```

Comment:
Again just match directly on `dynamic_template_params`.  There's no need for a separate `is_dynamic_template` - the false case _is_ the None arm and it gets rid of the "is dynamic template but params is None" can't-happen case.  (At least I think it can't happen.)